ui/job-view/KeyboardShortcuts.jsx

```diff
@@ -130,7 +130,8 @@ class KeyboardShortcuts extends React.Component {
     const { selectedJob } = this.props;
 
     if (selectedJob) {
-      this.$rootScope.$emit(thEvents.jobRetrigger, selectedJob);
+      window.dispatchEvent(new CustomEvent(
+        thEvents.jobRetrigger, { detail: { job: selectedJob } }));
```

Comment:
Prettier (https://prettier.io/playground/) suggests wrapping like this:

```js
    if (selectedJob) {
      window.dispatchEvent(
        new CustomEvent(thEvents.jobRetrigger, { detail: { job: selectedJob } })
      );
    }
```

...though happy either way :-)

Reply:
Sounds good to me.  Let's be pretty.  :)